py/mattak/backends/uproot/dataset.py

```diff
@@ -603,16 +606,17 @@ def rescale_adc(vbias : numpy.ndarray, adc : numpy.ndarray, Vref = 1.5) -> tuple
 
 
 def raw_calibrate(waveform_array : numpy.ndarray, vbias : numpy.ndarray, adc : numpy.ndarray,
-                        starting_window : Union[float, int]) -> numpy.ndarray:
+                  starting_window : Union[float, int],
+                  num_phys_samples : int = 4096, num_wf_samples : int = 2048) -> numpy.ndarray:
```

Comment:
should these default to the constants?

Reply:
These are no class functions. If we want to use contants here we have to define global variables

Reply:
you can though, because you defined the constants as class attributes (they're not instance attributes) so they should be accessible without an instance as mattak.AbstractDataset.foo ?